### PR TITLE
diversity: replaced oppressive language 

### PR DESCRIPTION
**Pull request by parosen**
All references that the community has identified as offensive and a relic of past oppression have been replaced with neutral definitions of the images that comprise an interferogram through the entire repository.  Testing was done on stripmapApp.xml

Using this version of isce2 requires the changes to all input xml files.

### CODE REVIEW

**Comment by hfattahi** [comment]
@yunjunz  I tried to fix a conflict in cuDenseOffsets.py. Mainly accepting your commits from last merged PR and changing slightly to match @parosen 's  PR:

https://github.com/isce-framework/isce2/blob/master/contrib/PyCuAmpcor/examples/cuDenseOffsets.py

Can you please give it a try and verify if things are not broken for you? Thanks

**Comment by hfattahi** [comment]
@dbekaert please verify if TOPS processing is working smoothly and aria pipelines not broken. Thanks

**Comment by yunjunz** [comment]
Hi @hfattahi, verifying the cuDenseOffsets.py requires re-install isce2 with GPU support, maybe @lijun99 could lend a hand for this?

Without re-install, the current run gives an error `AttributeError: 'PyCuAmpcor.PyCuAmpcor' object has no attribute 'referenceImageName'`, which is actually included in this PR.

**Comment by hfattahi** [comment]
> Hi @hfattahi, verifying the cuDenseOffsets.py requires re-install isce2 with GPU support on kamb, maybe @lijun99 could lend a hand for this?
> 
> Without re-install, the current run gives an error `AttributeError: 'PyCuAmpcor.PyCuAmpcor' object has no attribute 'referenceImageName'`, which is actually included in this PR.

Thank you @yunjunz . It will be great if @lijun99 or @rtburns-jpl could give it a try.

**Comment by lijun99** [comment]
Thanks for changing the names for PyCuAmpcor as well! I can test it on kamb. Or @yunjunz, you could try to compile your own copy. There is a guide on geo.gps for the scons method.

**Review by yunjunz — APPROVED** [review]
@hfattahi `contrib/PyCuAmpcor/examples/cuDenseOffsets.py` works. Thank you @lijun99 for the compilation!

**Comment by piyushrpt** [comment]
This branch compiles with cmake without any changes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ git reset --hard origin/master
 
 ```
 git checkout master
-(potentially update your local master against upstream, as described above)
+(potentially update your local reference against upstream, as described above)
```
Comment (rtburns-jpl) [inline CONTRIBUTING.md:61]: Would probably leave this unchanged since the git branch names are unaffected

**Review by rtburns-jpl — COMMENTED** [review]
Everything compiles and imports properly for me. Will defer to others as to the functionality of the python processing workflows.

**Comment by piyushrpt** [comment]
topsApp.py worked for me out of the box. I did not try all the parameter combinations.

test_topsapp

**Review by piyushrpt — APPROVED** [review]

**Pull request merged**